docs/source/features/version_control/branch.rst

```diff
@@ -0,0 +1,105 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+ 
+########
+ Branch
+########
+
+Each dataset is created with a default branch ``main``. When getting a dataset through the SDK,
+its version is the latest commit of the default branch.
+
+The following sections will introduce the operations and precautions related to branches
+in the SDK. Of course, it is necessary to get a dataset first:
+
+.. code:: python
+
+   from graviti import Workspace
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.get(f"{DATASET_NAME}")
+
+.. note::
+   In most cases, a branch can be thought of as a named commit. But if there is a dataset that
+   has just been created and has no commit history, the commit id of its default branch is
+   ``None``.
+
+
+*****************
+ Create a Branch
+*****************
+
+SDK provides method :meth:`~graviti.manager.branch.BranchManager.create` to support creating a
+branch based on a commit:
+
+.. code:: python
+
+   dataset.branches.create(f"{BRANCH_NAME}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{BRANCH_NAME}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{COMMIT_ID}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{TAG_NAME}")
+
+.. note::
+   If no revision is specified, the created branch will be based on the current commit of the
+   dataset, which can be viewed by ``dataset.HEAD.commit_id``.
+
+.. warning::
+   It is not allowed to create a new branch based on the default branch with no commit history,
+   for example:
+
+   .. code:: python
+
+      dataset = ws.datasets.create("create_branch_demo")
+      dataset.branches.create("branch-1")
+
+***************
+ List Branches
+***************
+
+SDK provides method :meth:`~graviti.manager.branch.BranchManager.list` to support listing
+branches:
+
+.. code:: python
+
+   dataset.branches.list()
+
+**************
+ Get a Branch
+**************
+
+SDK provides method :meth:`~graviti.manager.branch.BranchManager.get` to support getting a branch:
+
+.. code:: python
+
+   dataset.branches.get(f"{BRANCH_NAME}")
```

Comment:
Do we need to tell users that they can get the sheet from `branch/draft/commit/tag`?

Reply:
It doesn't make much sense to say `branch/commit/tag`, because we prefer using `checkout`. `Draft` is necessary. But this is `version control`, all of this content should be added to `sheet management`.